### DISCUSSION TITLE
How do you say £1000 in Cockney slang?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#749] [Galaxy Trivia] [answered]
**Bonus:** What are other denominations called?

**Comment by SANKARAMDAS** [comment, marked as answer]
£1000 | Bag of sand

£1,000 is commonly referred to as a grand, or rarely in certain dialects as a "bag" (from the rhyming slang "Bag of Sand"), 

Denomination  |  Cockney rhyming slang
Penny | Kilkenny
£1 | Lost and found
£5 | Lady Godiva, Deep sea diver
£10 | Cock and Hen
£15 | Commodore
£20 | Bobby Moore
£25 | Pony
£50 | Bullseye
£100 | Ton
£500 | Monkey
£1000 | Bag of sand

**Reply by ParanoidUser** [reply]
It would be nice to format this table as well. 😉

**Comment by peterbaccaro** [comment]
£1000 in Cockney slang is referred to as a **bag of sand**.

Other denominations are:

<html>
<body>


Denomination | Cockney rhyming slang
-- | --
Penny | Kilkenny
£1 | Lost and found
£5 | Lady Godiva, Deep sea diver
£10 | Cock and Hen
£15 | Commodore
£20 | Bobby Moore
£25 | Pony
£50 | Bullseye
£100 | Ton
£500 | Monkey
£1000 | Bag of sand


</body>
</html>
